### DISCUSSION TITLE
How to add image with link on my github profile readme.md?

### DISCUSSION THREAD

**Discussion by DavidDivellla** [community/community#54141] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I want to create image with link on my github profile. How to edit my readme file?

**Comment by GithubRealFan** [comment, marked as answer]
Please add bellow line to your github profile readme file :
[![image](Image Link on Github)](Your Link Address)

**Reply by balqisaini** [reply]
cannot play the vid. Only img appear on the screen.. how to do?
